packages/xstate-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@xstate/react",
-  "version": "1.0.0-rc.3",
+  "version": "1.0.0-rc.6",
```

Comment:
this hasnt been published yet - I think it would be best to only bump versions when actually releasing them

sideways q: wasn't `rc.5` merged into master? i see a 3 -> 6 bump here, which looks weird and maybe we've lost some git history along the way?

Reply:
I think it was bumped out-of-band. Sorry about that.